dnf/conf/config.py

```diff
@@ -848,6 +849,15 @@ def _configure_from_options(self, opts):
                     msg = "Main config did not have a %s attr. before setopt"
                     logger.warning(msg, name)
 
+    def _adjust_conf_options(self):
+        """Adjust conf options interactions"""
+
+        skip_broken = self._get_option('skip_broken')
+        skip_broken_val = skip_broken._get()
```

Comment:
skip_broken is instance. I think that if you will ask directly, you will receive value, but by that way, you exactly see what you want and what you use. Anyway late I have to request priority, therefore at that point it could be not easy to understand what I did here.